system/scripts/npc/city/c_Klaipe.lua

```diff
@@ -1,216 +1,213 @@
+ 
+--- [Accessory Merchant]Ronesa ---
 addnpc(20104, "ETC_20150317_009197", "c_Klaipe", 269, -1, -611, 45, "npc_accessorymerchantronesa")
 function npc_accessorymerchantronesa()
-	msg("Alfonso_Select_1")
-	openshop("Klapeda_Accessory")
+    msg("Alfonso_Select_1")
+    openshop("Klapeda_Accessory")
 end
-
---- Indifferent Widow --- 
+ 
+--- Indifferent Widow ---
 addnpc(20114, "QUEST_20150317_000083", "c_Klaipe", -409, -1, -647, -47.29, "npc_indifferentwidow")
 function npc_indifferentwidow()
-	if select("TUTO_GIRL_basic_02", "!@#$TUTO_GIRL_select01#@!", "!@#$Auto_JongLyo#@!") ~= 1 then
-		return
-	end
-	if select("TUTO_GIRL_basic_03", "!@#$TUTO_GIRL_select02#@!", "!@#$Auto_JongLyo#@!") ~= 1 then
-		return
-	end
-	if select("TUTO_GIRL_basic_04", "!@#$TUTO_GIRL_select03#@!", "!@#$Auto_JongLyo#@!") ~= 1 then
-		return
-	end
-
-	msg("TUTO_GIRL_basic_05")
+    if select("TUTO_GIRL_basic_02", "!@#$TUTO_GIRL_select01#@!", "!@#$Auto_JongLyo#@!") ~= 1 then
+        return
+    end
+    if select("TUTO_GIRL_basic_03", "!@#$TUTO_GIRL_select02#@!", "!@#$Auto_JongLyo#@!") ~= 1 then
+        return
+    end
+    if select("TUTO_GIRL_basic_04", "!@#$TUTO_GIRL_select03#@!", "!@#$Auto_JongLyo#@!") ~= 1 then
+        return
+    end
+ 
+    msg("TUTO_GIRL_basic_05")
 end
-
---- [Templar Master]Knight Commander Uska --- 
+ 
+--- [Templar Master]Knight Commander Uska ---
 addnpc(20113, "ETC_20151224_019888", "c_Klaipe", -474, 149, 82, 360, "npc_dummy")
-
---- [Cleric Master] Rozalija --- 
+ 
+--- [Cleric Master] Rozalija ---
 addnpc(20112, "ETC_20150317_009207", "c_Klaipe", -409, 149, 174, 360, "npc_dummy")
-
---- Klaipeda Resident --- 
+ 
+--- Klaipeda Resident ---
 addnpc(20158, "QUEST_LV_0100_20150317_000413", "c_Klaipe", -9, 156, 131, -23.96, "npc_dummy")
-
---- Mother of a Soldier --- 
+ 
+--- Mother of a Soldier ---
 addnpc(20114, "QUEST_LV_0100_20150317_000423", "c_Klaipe", -60, 148, 42, 36.57, "npc_dummy")
-
---- [Priest Master] Boruble --- 
+ 
+--- [Priest Master] Boruble ---
 addnpc(57229, "ETC_20150317_009208", "c_Klaipe", -196, 149, 351, -45, "npc_dummy")
-
---- Bulletin Board --- 
+ 
+--- Bulletin Board ---
 addnpc(40090, "QUEST_20150317_000081", "c_Klaipe", -80, 148, 387, -15, "npc_dummy")
-
---- [Blacksmith] Zaras --- 
+ 
+--- [Blacksmith] Zaras ---
 addnpc(20105, "ETC_20150317_009199", "c_Klaipe", 600, -1, -83, 45, "npc_dummy")
-
---- [Krivis Master] Herkus --- 
+ 
+--- [Krivis Master] Herkus ---
 addnpc(57228, "ETC_20150317_009209", "c_Klaipe", -1108, 240, 512, 360, "npc_dummy")
-
---- Indifferent Widow --- 
-addnpc(20114, "QUEST_20150317_000083", "c_Klaipe", -409, -1, -647, -47.29, "npc_dummy")
-
---- Refugee --- 
+ 
+--- Refugee ---
 addnpc(20139, "QUEST_LV_0100_20150317_001418", "c_Klaipe", -60, 79, -446, -33.69, "npc_dummy")
-
---- [Market Manager] Logi --- 
+ 
+--- [Market Manager] Logi ---
 addnpc(20169, "ETC_20150317_009211", "c_Klaipe", 278, 79, 137, 45, "npc_dummy")
-
---- [Peltasta Master]Maria Leed --- 
+ 
```

Comment:
Without those useless spaces this diff would be much shorter^^

Reply:
I'll remove the spaces.